downstream/titles/aap-operator-installation/master.adoc

```diff
@@ -1,6 +1,7 @@
 :imagesdir: images
 :numbered:
 :toclevels: 1
+:context: operator-platform-doc
```

Comment:
Added context to `master.adoc` because one of the included assemblies has `_{context}` in its `id`